pom.xml

```diff
@@ -18,10 +19,6 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
     <fcrepo-java-client.version>0.2.1-SNAPSHOT</fcrepo-java-client.version>
-    <!--We do NOT want to rely on a SNAPSHOT version of fcrepo; however, the dependency conflict between the-->
```

Comment:
We should keep a modified version of this comment for the property: `fcrepo-java-client.version`.